dev-libs/hyphen/hyphen-2.8.8-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="ALTLinux hyphenation library"
+HOMEPAGE="http://hunspell.github.io/"
+SRC_URI="mirror://sourceforge/hunspell/${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1 MPL-1.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+RDEPEND="app-text/hunspell"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+"
+
+DOCS=( AUTHORS ChangeLog NEWS README{,.nonstandard,.hyphen,.compound} THANKS TODO )
+
+src_configure() {
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	docinto pdf
+	dodoc doc/*.pdf
+
+	rm -f "${ED}"usr/lib*/libhyphen.la
```

Comment:
Does this only apply with `-static-libs` use? You might need to add some `if` logic around this, and append `|| die`. Not sure if you should get the libdir with a variable too, or does this work in every case.

Or just use the default way of pruning these files,  
`find "${ED}" -name '*.la' -delete || die`